lib/github.atd

```diff
@@ -590,7 +600,7 @@ type repo_issue_event = {
   ?label: base_label option;
   ?assignee: linked_user option;
   ?assigner: linked_user option;
-  ?milestone: milestone option;
+  ?milestone: repo_issue_milestone option;
```

Comment:
Great... the examples on that page have full milestone objects but I confirm that, e.g. <https://api.github.com/repos/moby/moby/issues/34222/events>, has `milestone` with only a `title`.

This change breaks type compatibility. Could you please put it in a commit by itself? It doesn't seem to be related to adding repository search.